src/chat.rs

```diff
@@ -1521,9 +1675,10 @@ pub fn create_group_chat(
         ],
     )?;
 
-    let chat_id = sql::get_rowid(context, &context.sql, "chats", "grpid", grpid);
+    let row_id = sql::get_rowid(context, &context.sql, "chats", "grpid", grpid);
+    let chat_id = ChatId::new(row_id);
 
-    if chat_id != 0 {
+    if row_id != 0 {
```

Comment:
Could be `chat_id.is_unset()`

Reply:
Good shout.  `!chat_id.is_error()` though.